openpype/lib/vendor_bin_utils.py

```diff
@@ -86,6 +86,41 @@ def find_executable(executable):
     return None
 
 
+def create_hard_link(src_path, dst_path):
```

Comment:
Should this function be called `create_hardlink`? Since the name of the actual feature is "hardlink" without a space, right?